packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+
+### Changed
+
+- **BREAKING** remove redundant perDomainNetwork state ([#3989](https://github.com/MetaMask/controllers/pull/3989))
```

Comment:
I know it's just for us. But maybe let's add explanation of what the consumer should do to adapt.

Reply:
actually maybe its all too context dependent. Lets actually just drop `redundant` since its unclear what's redundant out of the context of the extension.
```suggestion
- **BREAKING** remove `perDomainNetwork` value in state. ([#3989](https://github.com/MetaMask/controllers/pull/3989))
```

Reply:
Would this maybe fit better under a `### Removed` heading?